lib/utils/resolve-error-location.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const anonymizeStacktracePaths = require('./anonymize-stacktrace-paths');
+const tokenizeException = require('./tokenize-exception');
+
+const resolveErrorLocation = (error) => {
+  const exceptionTokens = tokenizeException(error);
+
+  if (exceptionTokens.isUserError && error.code) return null;
+
+  if (!exceptionTokens.stack) return null;
+
+  const splittedStack = exceptionTokens.stack.split('\n');
+  if (splittedStack.length === 1 && error.code) return null;
+
+  const stacktraceLineRegex = /\s*at.*\((.*)\).?/;
+  const stacktracePaths = [];
+  for (const line of splittedStack) {
+    const match = stacktraceLineRegex.exec(line) || [];
+    const matchedPath = match[1];
+    if (matchedPath) {
+      stacktracePaths.push(matchedPath);
+      // Limited to maximum 5 lines in location
+      if (stacktracePaths.length > 4) {
+        break;
+      }
+    } else if (stacktracePaths.length) break;
+  }
+
+  if (!stacktracePaths.length) return null;
+
+  return anonymizeStacktracePaths(stacktracePaths);
```

Comment:
As one of the ideas  is to treat it as programmer error identifier (so we can easily then filter out duplicates), I'd:
- Return it as _string_ (simply via `.join("\n")`)
- On Windows system I'd replace `\` with `/`. So errors coming from windows match those from posix machines